jenkins-vars.yml

```diff
@@ -17,7 +17,7 @@ repo_vars:
   - PR_DOCKERHUB_IMAGE = 'lspipepr/kicad'
   - DIST_IMAGE = 'alpine'
   - DIST_TAG = 'edge'
```

Comment:
this should be 3.21, right?

Reply:
Is it even used ? Seems to work, might be good to remove it when we go around again.